doc/page01_info.md

```diff
@@ -185,19 +191,37 @@ $ ./output
 
 That's it, **EVE** is properly installed and ready to use.
 
+## Use in CMake
+
+Once installed, **EVE** may be consumed though its config-file **CMake** package. Simply find and
+link against **EVE's** **CMake** target, as you would any other **CMake** library, and then
+configure and build your **CMake** project.
+
+```cmake
+add_executable(use-eve main.cpp)
+find_package(eve CONFIG REQUIRED)
+target_link_libraries(use-eve PRIVATE eve::eve)
+```
+
+> If a custom installation prefix was used, ensure your **EVE** installation is within **CMake's**
+  search path with the use of the **CMake** variables **eve_ROOT** (CMake 3.12), **eve_DIR**, or
+  **CMAKE_PREFIX_PATH**.
+
 # Advanced options
 
 If you want to dig a bit further, you can pass additional options to `cmake` to
 activate additional features.
 
 | Option | Effect       | Target               |
 |--------|:-------------|:---------------------|
-| `EVE_BUILD_TEST`      | Enable unit tests for **EVE** (`ON` by default).       | `ninja unit`        |
-| `EVE_BUILD_BENCHMARKS`| Enable benchmark tests for **EVE** (`OFF` by default). | `ninja benchmarks`  |
-| `EVE_BUILD_RANDOM`    | Enable random tests for **EVE** (`OFF` by default).    | `ninja random`      |
-
-There is currently over 2000 tests, so compiling all unit tests may require a large machine or
-some time. We recommend compiling in parallel using `-j`.
+| `EVE_BUILD_TEST`      | Enable unit tests for **EVE** (`ON` by default).       | `unit`        |
+| `EVE_BUILD_BENCHMARKS`| Enable benchmark tests for **EVE** (`OFF` by default). | `benchmarks`  |
+| `EVE_BUILD_RANDOM`    | Enable random tests for **EVE** (`OFF` by default).    | `random`      |
```

Comment:
Didn't we had a `EVE_BUILD_INTEGRATION` entry at some point or did I dreamt too deep ?

Reply:
Yes, that option was added as part of #1336 to enable the integration tests that are now registered with CTest. Thanks for catching my not updating this documentation section to match! I'll do that.

Reply:
I added these docs and made an aggregated `integration` target to run the integration tests. This target was really just to have a target to put in the table, though ;)